File: ui/src/main/java/com/midtrans/sdk/corekit/models/snap/Installment.java
package com.midtrans.sdk.corekit.models.snap;

import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class Installment {
    private boolean isRequired;
    private Map<String, ArrayList<Integer>> terms;

    public boolean isRequired() {
        return isRequired;
    }

    public void setRequired(boolean required) {
        isRequired = required;
    }

    public Map<String, ArrayList<Integer>> getTerms() {
        return terms;
    }

    public void setTerms(Map<String, ArrayList<Integer>> terms) {
        this.terms = terms;
    }
}

File: ui/src/main/kotlin/com/midtrans/sdk/uikit/api/model/Installment.kt
package com.midtrans.sdk.uikit.api.model

import com.midtrans.sdk.corekit.api.model.Installment


open class Installment(
    isRequired: Boolean = false,
    terms: Map<String, List<Int>>? = null,
) : Installment(
    isRequired = isRequired,
    terms = terms
)
